src/lib/comms/Makefile

```diff
@@ -92,6 +92,10 @@ obj-networking-$(OIC)-extra-cflags += \
     -DOIC_SUPPORT_URL=$(OIC_SERVER_SUPPORT_URL) \
     -DOIC_DEVICE_NAME=$(OIC_SERVER_DEVICE_NAME)
 
+ifeq (y,$(OIC_SERVER_COMPAT_1_0))
+COMMON_CFLAGS += -DOIC_SERVER_COMPAT_1_0
```

Comment:
is this needed? AFAIU this goes into sol-generated.h or something like that, which is auto-included in all builds